src/controllers/commands/CommandController.cpp

```diff
@@ -938,6 +938,97 @@ void CommandController::initialize(Settings &, Paths &paths)
         return "";
     });
 
+    auto formatModsError = [](HelixGetModeratorsError error, QString message) {
+        using Error = HelixGetModeratorsError;
+
+        QString errorMessage = QString("Failed to get moderators: ");
+
+        switch (error)
+        {
+            case Error::Forwarded: {
+                errorMessage += message;
+            }
+            break;
+
+            case Error::UserMissingScope: {
+                errorMessage += "Missing required scope. "
+                                "Re-login with your "
+                                "account and try again.";
+            }
+            break;
+
+            case Error::UserNotAuthorized: {
+                errorMessage +=
+                    "Due to Twitch restrictions, this command can only be "
+                    "used by moderators. To see the list of mods you must "
+                    "use the Twitch website.";
+            }
+            break;
+
+            case Error::Unknown: {
+                errorMessage += "An unknown error has occurred.";
+            }
+            break;
+        }
+        return errorMessage;
+    };
+
+    this->registerCommand(
+        "/mods",
+        [formatModsError](const QStringList &words, auto channel) -> QString {
+            switch (getSettings()->helixTimegateModerators.getValue())
+            {
+                case HelixTimegateOverride::Timegate: {
+                    if (areIRCCommandsStillAvailable())
+                    {
+                        return useIRCCommand(words);
+                    }
+                }
+                break;
+
+                case HelixTimegateOverride::AlwaysUseIRC: {
+                    return useIRCCommand(words);
+                }
+                break;
+                case HelixTimegateOverride::AlwaysUseHelix: {
+                    // Fall through to helix logic
+                }
+                break;
+            }
+
+            auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get());
+
+            if (twitchChannel == nullptr)
+            {
+                channel->addMessage(makeSystemMessage(
+                    "The /mods command only works in Twitch Channels"));
+                return "";
+            }
+
+            getHelix()->getModerators(
+                twitchChannel->roomId(),
+                [channel, twitchChannel](auto result) {
+                    QStringList list;
+                    for (auto it = result.begin(); it != result.end(); it++)
+                    {
+                        list << it->userName;
+                    }
+
+                    list.sort(Qt::CaseInsensitive);
+
+                    MessageBuilder builder;
+                    TwitchMessageBuilder::listOfUsersSystemMessage(
+                        "The moderators of this channel are ", list,
```

Comment:
```suggestion
                        "The moderators of this channel are", list,
```
before:
![image](https://user-images.githubusercontent.com/41973452/200047301-99fd81f4-3492-47ca-a0e7-40c784392ad4.png)
after:
![image](https://user-images.githubusercontent.com/41973452/200047547-2268118b-2de8-460a-ac14-58b2c18c2323.png)